main.go

```diff
@@ -165,54 +174,75 @@ func mainE(config Configs) error {
 
 	fmt.Println()
 
-	log.Infof("Export APKs:")
+	log.Infof("Export Artifacts:")
 	fmt.Println()
 
-	apks, err := getArtifacts(gradleProject, started, config.APKPathPattern, false)
+	appPatterns := append(strings.Split(config.AppPathPattern, "\n"), config.APKPathPattern)
+	appPatterns = sliceutil.UniqueStringSlice(appPatterns) // we still need to add the deprecated APKPathPattern which could cause a pattern duplication in the list
+	artifacts, err := getArtifacts(gradleProject, started, appPatterns, false)
 	if err != nil {
-		return fmt.Errorf("failed to find apks, error: %v", err)
+		return fmt.Errorf("failed to find artifacts, error: %v", err)
 	}
 
-	if len(apks) == 0 {
-		log.Warnf("No apks found with pattern: %s", config.APKPathPattern)
-		log.Warnf("If you have changed default APK export path in your gradle files then you might need to change APKPathPattern accordingly.")
+	if len(artifacts) == 0 {
+		log.Warnf("No artifacts found with patterns: %s", strings.Join(appPatterns, ", "))
+		log.Warnf("If you have changed default APK, AAB export path in your gradle files then you might need to change AppPathPattern accordingly.")
 		return nil
 	}
 
-	exportedArtifactPaths, err := exportArtifacts(apks, config.DeployDir)
+	exportedArtifactPaths, err := exportArtifacts(artifacts, config.DeployDir)
 	if err != nil {
 		return fmt.Errorf("Failed to export artifact: %v", err)
 	}
 
+	log.Debugf("Found artifacts:\n" + strings.Join(exportedArtifactPaths, "\n"))
+
+	// Remove the wrong build types
+	i := 0
+	for _, path := range exportedArtifactPaths {
```

Comment:
why do not you remove the wrong patterns? you could also notify the user about a wrong pattern if any.

Reply:
Sadly this is not an option, because it would work if the pattern would look like this:
```
dir/*.apk => in this case I can check the extension
```

But if the pattern looks like this:
```
/dir/* => I could not filter the patterns, just the found artifacts.
```